### PR TITLE
Add Papers Please Projects to Identity Team

### PR DESCRIPTION
**Pull request by johnsyweb**
#### Context

We have PRs awaiting review that aren't getting announced in `#identity-dev`.

#### Change

Add the four Papers Please repositories to our list

### CODE REVIEW

File: config/envato.yml [hunk 1]
```diff
@@ -131,6 +131,10 @@ identity:
     - identity-admin
     - identity-infrastructure
     - identity-team
+    - papers-please
+    - papers-please-pilot
+    - papers-please-spike
```
Comment (dferdian) [inline config/envato.yml]: This two `papers-please-pilot` and `papers-please-spike` are from the previous round of Papers Please plan which quite different from the plan we have now, so probably we don't need it

We might reuse `papers-please` repo for the implementation

Comment (johnsyweb) [reply config/envato.yml]: OK. Should we archive those repositories?

Comment (dferdian) [reply config/envato.yml]: Yep, I'll archive those repo, thanks

Comment (johnsyweb) [reply config/envato.yml]: 6d671be addresses this comment.

Comment (dferdian) [reply config/envato.yml]: I have archieve papers-please-pilot and papers-please-spike repo

**Pull request merged**